### ISSUE TITLE
Sign in to confirm not bot

### ISSUE THREAD

**Issue by Jestre** [kieraneglin/pinchflat#293]
I've been using Pinchflat for a month or so now, and while watching some videos this morning, I noticed others were missing.  Checking the console, I see that several have not downloaded, despite being several weeks old:

![SCR-20240617-kmux|350](https://github.com/kieraneglin/pinchflat/assets/2609516/3e797d9d-562b-4e31-8301-887fd5f95fa5)

I told it to download pending files, but when they still didn't show up, I checked the logs and I see the following:

`[youtube] 8v4R-bMWS_k: Sign in to confirm you’re not a bot. This helps protect our community. Learn more`

I am not using any of the fast indexing modes (perhaps I should be), but it looks like YT have cut me off.  I have only five sources (a total of 69 videos downloaded) so it's not like I'm trying to duplicate YT. 

Anything I can do, short of change IP on the server running this?

**Comment by Jestre** [comment]
No, I've actually configured very little in there, just added the five sources and for a few, the start date from which to download.  Beyond that, I've not enabled or changed anything, though I see a lot has been added over the past few months.

**Comment by silkyclouds** [comment]
Feb 18th 2025 and I have the same issue. So, there is no way to fix this ? I can't download anything anymore atm.

**Comment by MrPistachios** [comment]
I got hit with this two, also now I cant watch youtube without being logged in while on my network rip

**Comment by LinFor** [comment]
Have the same issue with different sources since 2025-03-15.

**Comment by Behv730** [comment]
same
